frontend/src/queries/nodes/DataTable/DataTable.tsx

```diff
@@ -439,7 +439,7 @@ export function DataTable({ uniqueKey, query, setQuery, context, cachedResults }
     return (
         <BindLogic logic={dataTableLogic} props={dataTableLogicProps}>
             <BindLogic logic={dataNodeLogic} props={dataNodeLogicProps}>
-                <div className="relative w-full flex flex-col gap-4 flex-1">
+                <div className="relative w-full flex flex-col gap-4 flex-1 h-full">
```

Comment:
Hard to know where some of the queries stuff gets affected but happy that it doesn't cause any related snapshot changes